addons/rating/static/src/models/thread_preview_view.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+
+import { patchRecordMethods } from '@mail/model/model_core';
+// ensure the model definition is loaded before the patch
+import '@mail/models/thread_preview_view';
+
+patchRecordMethods('ThreadPreviewView', {
+    /**
+     * @override
+     */
+    _computeIsEmpty() {
+        const isEmpty = this._super();
+        const isRating = this.thread.lastMessage && this.thread.lastMessage.rating;
```

Comment:
make this value a computed field by itself, and also use it as condition in the template